### ISSUE TITLE
Cytoscape svg not rendering images 

### ISSUE THREAD

**Issue by KevinZhou1** [kinimesi/cytoscape-svg#1]
Hello, when we use cytoscape svg, not all images are being rendered for each node. Has this issue been seen before?

**Comment by kinimesi** [comment]
Not rendered in the exported SVG or in Cytoscape canvas?

It shouldn't interfere with Cytoscape canvas. Can you provide a reproducible demo, please?

**Comment by KevinZhou1** [comment]
It is not rendered in the exported svg.  We are doing the following to save the svg:


![SELECTED_PROJECT](https://user-images.githubusercontent.com/12479958/60520609-b322eb80-9cb3-11e9-9f41-2b7e5d49b198.png)
![TOO_MANY_PROJECT](https://user-images.githubusercontent.com/12479958/60520651-c6ce5200-9cb3-11e9-90d2-7df6fea2d4e3.png)
![TERMINAL_PROJECT](https://user-images.githubusercontent.com/12479958/60520676-cfbf2380-9cb3-11e9-8e8c-10ba1a3433ac.png)
![DEFAULT_PROJECT](https://user-images.githubusercontent.com/12479958/60520722-e36a8a00-9cb3-11e9-94e1-242ee3874f43.png)
 `    var svg = document.createElement("a");
    document.body.appendChild(svg);
    svg.style = "display: none";
    svg.href = URL.createObjectURL(new Blob([this.cy.svg({full: true, scale: 0.5})], {type: 'image/svg+xml'}));
    svg.download = this.state.downloadName;
    svg.click();
    window.URL.revokeObjectURL(svg.href);
    svg.parentNode.removeChild(svg);`


It is working but is not displaying the images correctly on some of the nodes, specifically we have different node images that represent whether a node is selected, terminal, or have many dependencies. Although we can see it on the canvas, we cannot see it when we export it as svg.  


The first two icons (png) are not  working (highlighted yellow and have the kg symbol), the bottom two are appearing in the svg. I have also tried switching theses nodes images around no matter what, I could not get them to appear. Let me know if there is anything else I can provide.

**Comment by kinimesi** [comment]
It should be fixed with https://github.com/kinimesi/canvas2svg/commit/29d64b69dd91a9928ce02a0ecc20e391aaeaea15 and https://github.com/kinimesi/cytoscape-svg/commit/96da5633e4c0aededab4aae759172aad65c9412d

Can you test it with the master branch and verify please?

**Comment by KevinZhou1** [comment]
Thank you so much ! It is working now. My team greatly appreciates your efforts

**Comment by KevinZhou1** [comment]
When can npm be updated to include the latest code? In order to fix the bug, I have to manually clone and copy it to the node_modules folder since npm still fetches the older one

**Comment by kinimesi** [comment]
Released as v0.2.0

PS: you can directly add the GitHub repository to your package.json as well instead of manual copying. Related NPM [doc](https://docs.npmjs.com/files/package.json#github-urls).
```json
{  
"dependencies": {
    "cytoscape-svg": "kinimesi/cytoscape-svg#master"
 }
}
```
